src/element/transformHandles.ts

```diff
@@ -248,12 +252,31 @@ export const getTransformHandles = (
   } else if (isTextElement(element)) {
     omitSides = OMIT_SIDES_FOR_TEXT_ELEMENT;
   }
-
+  const dashedLineMargin = isLinearElement(element) ? 12 : 4;
```

Comment:
is the `12` in any way related to the `8` we use in rendering? Also, I know we'll be removing it soon, but I'd use constants for these so we can later easily find all occurrences to replace.

Reply:
I guess you are referring to `renderSelectionBorder` ? Nope I don't think so. 
Both have their own calculations so should be fine.